### PR TITLE
Draft ambassadors proposal

### PR DESCRIPTION
**Pull request by tghosth**

### CODE REVIEW

**Review by northdpole — APPROVED** [review]

File: proposals/2022-ambassadors.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+To qualify as an OWASP ambassador, the person should commit to going to four non-OWASP events per year with the aim of advocating for OWASP and track this for audit on renewal. This would be a purely volunteer position and new applicants would have to be able to demonstrate previous OWASP involvement as well as having OWASP membership.
+
+In return, the ambassador would receive:
+* Specific recognition and thanks
+* Supplies of very basic/inexpensive OWASP swag such as stickers or pens to give out
+* Some dedicated OWASP apparel to make them stand out.
+* Coordination with the local chapter
+
+### OWASP stall
```
Comment (joubin) [inline proposals/2022-ambassadors.md:31]: Should this be staff?

File: proposals/2022-ambassadors.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* A plan for who would staff the table (could be coordinated with the local chapter)
+* A quote for having the additional SWAG below prepared locally rather than having to ship it from elsewhere.
+
+They could then be provided with extra resources (in addition to those noted above) including:
+* OWASP tablecloth
+* Larger information flyers about OWASP
+* Some sort of additional swag (like a unique t-shirt) to be dedicated for people who become OWASP members on the spot
+
+
+## Budget required
```
Comment (joubin) [inline proposals/2022-ambassadors.md:47]: I would think this is where per year, the budget is set by the Board. 

- The money is set at the start of the year and it's up to the out reach committee to plan and prioritize to list and identify different people and approaches. 
- It is up to the committee to choose if a select ambassador will attend or if there is a trusted member of a local chapter near the conference 
- It is up to the committee to say if a conference should have a booth, if we are sending people and engaging with the conference to see if we can obtain booths at lower costs

File: proposals/2022-ambassadors.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Proposal Summary
+
+Creation of an OWASP ambassador role who would attend meetups and conferences and provide more in-depth information about getting into OWASP. 
+
+## Proposal Details
+
+To qualify as an OWASP ambassador, the person should commit to going to four non-OWASP events per year with the aim of advocating for OWASP and track this for audit on renewal. This would be a purely volunteer position and new applicants would have to be able to demonstrate previous OWASP involvement as well as having OWASP membership.
+
+In return, the ambassador would receive:
```
Comment (joubin) [inline proposals/2022-ambassadors.md:25]: Provide a code of conduct for who is representing the org.

**Comment by joubin** [comment]
Some other risks to consider. 

- I think many people are gonna want to do this while I understand that we can only trust a very few to start. Consider a process where you can have KPIs, questions, qualifiers to approval additional people where you aren't just selecting favorites or excluding people
- Tell us how you won't impact the staff or how we can operationalize this so that the scalability of this is not 1:1 tied to staff availability

**Comment by joubin** [comment]
- Proposal 
  - The board allocates a dollar amount per year ear marked for the outreach committee to attend conferences and events as they see fit
  - The budget is allocated and approved based on KPIs created by the committee 
  - The KPIs are measured and the allocations are allocated over time
  - The committee should try to utilize OWASP relationships in obtaining free attendance to booths and possible support for conferences

File: proposals/2022-ambassadors.md (hunk 1 above)
Comment (tghosth) [reply proposals/2022-ambassadors.md:31]: No but I will changed the wording to clarify

Comment (tghosth) [reply proposals/2022-ambassadors.md:31]: No but I will change the wording to clarify

File: proposals/2022-ambassadors.md (hunk 3 above)
Comment (tghosth) [reply proposals/2022-ambassadors.md:25]: Good point, adding a clarification. Although, an OWASP member is implicitly agreeing to comply with a code of conduct through their membership, no?

File: proposals/2022-ambassadors.md (hunk 2 above)
Comment (tghosth) [reply proposals/2022-ambassadors.md:47]: Bullet 1 - I added this just below and included a point on metrics.
Bullet 2 - I dislike the idea of ambassadors incurring travel expenses to attend an event except potentially for the very largest events which in turn should only be the most experienced/highly regarded ambassadors.
Bullet 3 - Added some clarification below

**Pull request merged**